spec/responses/stash_api/helpers.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+module Helpers
+
```

Comment:
Nice. Thanks for placing all of this in a helper. It will make future API tests easier to write.